requirements.txt

```diff
@@ -1,3 +1,6 @@
 catalyst
```

Comment:
could you please specify catalyst version? for 20.1 for example

Reply:
Done